flexget/api/core/authentication.py

```diff
@@ -123,7 +122,7 @@ def post(self, session=None):
 @auth_api.route('/logout/')
 class LogoutAPI(APIResource):
     @api.response(200, 'Logout successful', model=base_message_schema)
-    def post(self, session=None):
+    def post(self, session: None = None) -> Response:
```

Comment:
type shouldn't be `None`, it's `Session`

Reply:
`session` gets injected by the `@with_session` decorator, which is not applied to this method (and all others annotated as `None`). Therefore it's really `None` in this case.

Even if I'd managed to correctly annotate the `with_session()` method, these instances would have to be `Optional[Session]` due to the `None` initialisation